{"@timestamp":"2025-01-04T03:39:51.536Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:40:21.541Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:40:26.540Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:43:31.558Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:43:36.559Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:45:46.574Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:45:56.575Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:46:11.576Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:46:56.582Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:51:46.621Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:51:56.622Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:52:46.628Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:55:56.651Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:56:01.652Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T03:59:56.678Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:04:06.705Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:07:36.732Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:11:41.758Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:12:51.767Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:16:56.792Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:20:51.820Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:24:56.846Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:56:40.945Z","caller":"sqlx/utils.go:142","content":"Error on getting sql instance of tcp(172.17.0.2:3306)/mall_product?parseTime=true&loc=Local&charset=utf8mb4&collation=utf8mb4_unicode_ci: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error","span":"7431bc6254f538ef","trace":"c7be99265426af847c0f3594316b509f"}
{"@timestamp":"2025-01-04T04:56:40.945Z","caller":"sqlx/utils.go:142","content":"Error on getting sql instance of tcp(172.17.0.2:3306)/mall_product?parseTime=true&loc=Local&charset=utf8mb4&collation=utf8mb4_unicode_ci: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error","span":"0cdc962981d8be0b","trace":"0410842e93521ebd15b23942f6990de4"}
{"@timestamp":"2025-01-04T04:56:40.945Z","caller":"logic/listproductslogic.go:40","content":"Failed to get products count: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error"}
{"@timestamp":"2025-01-04T04:56:40.945Z","caller":"logic/listproductslogic.go:40","content":"Failed to get products count: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error"}
{"@timestamp":"2025-01-04T04:56:43.098Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:56:45.489Z","caller":"sqlx/utils.go:142","content":"Error on getting sql instance of tcp(172.17.0.2:3306)/mall_product?parseTime=true&loc=Local&charset=utf8mb4&collation=utf8mb4_unicode_ci: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error","span":"adb739014b136370","trace":"06d256d83052a726bf9ec2bf80a8633a"}
{"@timestamp":"2025-01-04T04:56:45.489Z","caller":"sqlx/utils.go:142","content":"Error on getting sql instance of tcp(172.17.0.2:3306)/mall_product?parseTime=true&loc=Local&charset=utf8mb4&collation=utf8mb4_unicode_ci: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error","span":"c34f48baedc3806d","trace":"77604104e176b4f43c2b3ea191bc9ecf"}
{"@timestamp":"2025-01-04T04:56:45.489Z","caller":"logic/listproductslogic.go:40","content":"Failed to get products count: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error"}
{"@timestamp":"2025-01-04T04:56:45.489Z","caller":"logic/listproductslogic.go:40","content":"Failed to get products count: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error"}
{"@timestamp":"2025-01-04T04:56:48.099Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:56:53.100Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:56:53.489Z","caller":"sqlx/utils.go:142","content":"Error on getting sql instance of tcp(172.17.0.2:3306)/mall_product?parseTime=true&loc=Local&charset=utf8mb4&collation=utf8mb4_unicode_ci: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error","span":"2933aabd78701956","trace":"f82e8e491d3626faa80869dc7abfb0f0"}
{"@timestamp":"2025-01-04T04:56:53.489Z","caller":"logic/listproductslogic.go:40","content":"Failed to get products count: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error"}
{"@timestamp":"2025-01-04T04:56:53.489Z","caller":"sqlx/utils.go:142","content":"Error on getting sql instance of tcp(172.17.0.2:3306)/mall_product?parseTime=true&loc=Local&charset=utf8mb4&collation=utf8mb4_unicode_ci: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error","span":"230186eed75b2b63","trace":"63741b925bd0918e866fdd5863af9e5b"}
{"@timestamp":"2025-01-04T04:56:53.489Z","caller":"logic/listproductslogic.go:40","content":"Failed to get products count: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error"}
{"@timestamp":"2025-01-04T04:56:58.100Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:57:43.105Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T04:58:33.109Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T05:01:48.133Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T05:05:48.163Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T05:09:58.191Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T05:44:48.436Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T05:45:23.441Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T05:48:48.468Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T05:53:58.506Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T05:56:48.529Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T05:57:18.531Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:05:08.590Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:05:13.591Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:11:18.636Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:12:08.642Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:14:03.658Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:16:13.676Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:24:18.735Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:30:28.781Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:30:48.782Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:34:53.810Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:38:53.838Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:41:48.860Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:42:53.867Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:51:28.935Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:54:08.954Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:56:58.973Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:59:08.990Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T06:59:33.993Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T09:18:14.966Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:09:50.447Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:09:50.789Z","caller":"sqlx/utils.go:142","content":"Error on getting sql instance of tcp(172.17.0.2:3306)/mall_product?parseTime=true&loc=Local&charset=utf8mb4&collation=utf8mb4_unicode_ci: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error","span":"15fd244de7e558c4","trace":"ed2563d966f4a5b432a982f0ca83aa21"}
{"@timestamp":"2025-01-04T11:09:50.789Z","caller":"logic/listproductslogic.go:40","content":"Failed to get products count: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error"}
{"@timestamp":"2025-01-04T11:09:50.789Z","caller":"sqlx/utils.go:142","content":"Error on getting sql instance of tcp(172.17.0.2:3306)/mall_product?parseTime=true&loc=Local&charset=utf8mb4&collation=utf8mb4_unicode_ci: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error","span":"d54291a669b586c2","trace":"bd16478d207853a2eaae85f325f37e7d"}
{"@timestamp":"2025-01-04T11:09:50.789Z","caller":"logic/listproductslogic.go:40","content":"Failed to get products count: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error"}
{"@timestamp":"2025-01-04T11:09:55.447Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:09:57.157Z","caller":"sqlx/utils.go:142","content":"Error on getting sql instance of tcp(172.17.0.2:3306)/mall_product?parseTime=true&loc=Local&charset=utf8mb4&collation=utf8mb4_unicode_ci: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error","span":"bd54635699766ac1","trace":"20776b28cce7ac1222c40d4c27a4d82f"}
{"@timestamp":"2025-01-04T11:09:57.157Z","caller":"logic/listproductslogic.go:40","content":"Failed to get products count: dial tcp 172.17.0.2:3306: connect: no route to host","level":"error"}
{"@timestamp":"2025-01-04T11:10:00.448Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:10:40.453Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:13:45.471Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:13:55.472Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:15:20.484Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:15:25.485Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:15:35.487Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:15:40.487Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:17:10.497Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:20:45.522Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T11:43:40.684Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:24:30.969Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:25:20.974Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:25:30.975Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:25:35.976Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:29:26.006Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:29:41.008Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:32:56.030Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:33:16.033Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:36:21.051Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:36:51.055Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:39:41.075Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:41:26.085Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:43:36.101Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T12:43:41.102Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:02:41.233Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:02:46.234Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:02:56.235Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:03:41.242Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:03:46.243Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:13:16.324Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:13:31.326Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:22:21.380Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:23:01.384Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:23:11.385Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:31:11.438Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:32:06.445Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:32:11.446Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:42:51.528Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:44:16.538Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:44:36.541Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:48:51.569Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:48:56.569Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:53:01.597Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:53:06.598Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T13:56:36.623Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:00:36.650Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:04:46.686Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:08:46.713Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:12:46.738Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:16:51.767Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:20:56.791Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:21:01.792Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:25:06.824Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:28:31.848Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:32:36.876Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:36:41.905Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:40:41.935Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:44:51.960Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:48:56.988Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:53:02.015Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:53:07.015Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T14:56:32.041Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T15:00:32.070Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T15:04:37.098Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T15:08:42.125Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-04T15:12:42.153Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T00:42:06.638Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T00:51:26.695Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T00:51:46.697Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T00:52:56.705Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T00:53:36.710Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T01:12:26.837Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T01:13:36.843Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T01:32:26.978Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T01:32:36.979Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T01:43:52.063Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T01:45:22.074Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T01:47:57.092Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:19:41.307Z","caller":"zrpc/server.go:90","content":"context deadline exceeded","level":"error"}
{"@timestamp":"2025-01-05T02:23:24.957Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:24:49.967Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:24:59.968Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:25:04.969Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:25:14.971Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:25:34.973Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:28:55.000Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:32:55.026Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:36:25.050Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:39:05.070Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:39:15.071Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:39:20.072Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:42:35.095Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:46:35.122Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:47:25.128Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T02:47:45.130Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:06:15.262Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:06:25.263Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:06:35.264Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:08:10.277Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:08:15.277Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:08:20.278Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:08:25.278Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:11:45.300Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:15:45.330Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:19:55.358Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:23:50.384Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:27:55.410Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:31:55.436Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:36:00.464Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:40:10.491Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:43:35.516Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T03:47:40.542Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:00:19.267Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:00:24.268Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:00:29.268Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:00:34.268Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:03:54.291Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:07:54.321Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:10:44.343Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:10:54.344Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:11:04.346Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:11:09.346Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:13:59.366Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:17:34.392Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:17:59.395Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:18:04.395Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:18:09.395Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:18:14.396Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:18:19.396Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:21:59.423Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T04:25:24.445Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T09:56:35.951Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"0c49f96714d1de3d","trace":"a1d5c5c64e2986a316fc927ec8561ec9"}
{"@timestamp":"2025-01-05T09:56:36.081Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"85c48618238219c4","trace":"2dbe0a9b0b621e5a04d5912b7011a7b6"}
{"@timestamp":"2025-01-05T09:56:38.999Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T10:00:53.004Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"6cd5b8a996080a0b","trace":"3346fd0b6428ba8a0196d6421f1b825b"}
{"@timestamp":"2025-01-05T10:00:53.118Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"51c371cb6afe0246","trace":"4cabe3cb9fa958a6f7df660c5b46b5e9"}
{"@timestamp":"2025-01-05T10:00:54.030Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T10:01:02.676Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"b8e891a808e892ff","trace":"68754b332d7557731d413e49eeeb6731"}
{"@timestamp":"2025-01-05T10:01:02.790Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"db7ec3711bd08cd2","trace":"aa5e9710071086cabcaa05fc94db7182"}
{"@timestamp":"2025-01-05T10:01:04.031Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T10:01:07.839Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"494b58787f5758bf","trace":"287096fa9704873c355ab00c78ec026d"}
{"@timestamp":"2025-01-05T10:01:07.948Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"fccb0097564b111a","trace":"fd41603af3dc5932896b85362feb0f38"}
{"@timestamp":"2025-01-05T10:01:09.032Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T10:01:14.165Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"a18a54cc1956038a","trace":"39a8e2b472546929b1f205a091b91464"}
{"@timestamp":"2025-01-05T10:01:15.870Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"7d2c5734310d28be","trace":"feec512fdc99c37fc12024397f9c311d"}
{"@timestamp":"2025-01-05T10:01:15.985Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"2219b5e9463adf0a","trace":"98f83620cd6cf21f9903b024c270519c"}
{"@timestamp":"2025-01-05T10:01:19.033Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T10:58:36.535Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"bd7a6cf268a919ef","trace":"99b731fb4bfd18d9fbc38baa7be6aa20"}
{"@timestamp":"2025-01-05T10:58:37.117Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"26e37d58b686637e","trace":"66aae4f40b562aef15cc7793f96c9b26"}
{"@timestamp":"2025-01-05T10:58:39.433Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T10:58:43.457Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"3adec9af06378f34","trace":"0f627cec52ed07e6683b4bb56092d014"}
{"@timestamp":"2025-01-05T10:58:44.030Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"b5c0d77861933ec9","trace":"23b6b2b578e243c79af52a4ac70f03f8"}
{"@timestamp":"2025-01-05T10:58:44.434Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T10:58:49.445Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T10:58:49.804Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"5e7906780a325f25","trace":"861315925c16ce85baf1bb5a5c4ade46"}
{"@timestamp":"2025-01-05T10:58:51.886Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"d1171f91e3c4c875","trace":"8bc4dcc6eca560ce04b809bd36e59338"}
{"@timestamp":"2025-01-05T10:58:52.459Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"cc03760890b86e38","trace":"5bfa43cb2280ba166ec2f1909e182be9"}
{"@timestamp":"2025-01-05T10:58:54.446Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:02:57.594Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"11728e596cdea2ed","trace":"eb7e1f83e4cc3a1a9b0d0359b06e902d"}
{"@timestamp":"2025-01-05T11:02:57.710Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"8b8a8c4821588278","trace":"a4c18b9512ff442dd29a7417ea68906c"}
{"@timestamp":"2025-01-05T11:02:59.474Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:03:00.696Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"2e0f3886a8e57571","trace":"7631c2bad2e42193d28ecaede84602ee"}
{"@timestamp":"2025-01-05T11:03:00.798Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"0a56391341e7f562","trace":"2cf582d67d6f7389bf145d0fc15e73c1"}
{"@timestamp":"2025-01-05T11:03:04.475Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:03:07.280Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"82f18f4e940f37c0","trace":"8ec6d5c75b11c75de3e4e92a9fd5136e"}
{"@timestamp":"2025-01-05T11:03:07.384Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"546b36151c9e608c","trace":"74071569a3df89ae1281809004957a55"}
{"@timestamp":"2025-01-05T11:03:09.476Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:06:31.113Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"f3c0d0058aa80b91","trace":"449f0b5cacec580814092866fe255ea1"}
{"@timestamp":"2025-01-05T11:06:31.245Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"ae73ff6405098211","trace":"24f46515113ac80b3e018a69b2892d9c"}
{"@timestamp":"2025-01-05T11:06:31.356Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"0781d8be6e50c9cb","trace":"71be6efcebec2a5d72a213a95eeaf45a"}
{"@timestamp":"2025-01-05T11:06:31.468Z","caller":"sqlx/utils.go:147","content":"stmt: select `id`,`name`,`brief`,`description`,`category_id`,`brand`,`images`,`price`,`sales`,`status`,`created_at`,`updated_at` from `categories`, error: Error 1054 (42S22): Unknown column 'brief' in 'field list'","level":"error","span":"ef05c30a9e698ef9","trace":"8ee17788ce4c1022a75427b981088914"}
{"@timestamp":"2025-01-05T11:06:34.498Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:09:42.503Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:09:47.503Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:09:52.504Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:09:57.504Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:10:02.505Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:10:07.506Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:10:32.509Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
{"@timestamp":"2025-01-05T11:10:37.510Z","caller":"trace/agent.go:148","content":"[otel] error: Post \"http://127.0.0.1:14268/api/traces\": dial tcp 127.0.0.1:14268: connect: connection refused","level":"error"}
